core/cmd/run_stream_cmd.go

```diff
@@ -99,7 +98,7 @@ func runMetricsAndProfiler(ctx context.Context, cfg *config.Config) error {
 	return nil
 }
 
-func runServer(cfg *config.Config) error {
+func runStreamMode(cfg *config.Config) error {
```

Comment:
Personally I would like to see the `cmd` package only deal with registering (sub)commands and arguments, loading configuration on start and call the appropriate function in the `xchain` or `node` package. That means this function should be moved to the `node` package and the caller should only have to do `stream.RunNode(cfg)` (if we rename the `node` package to `stream`)

Reply:
there is some extra setup in this function before calling Execute, it needs to be futher refactored, in this PR I'm moving files around mostly, so I don't want to introduce too drastic change.